core/src/main/resources/application.yml

```diff
@@ -1,6 +1,33 @@
 micronaut:
     application:
         name: core
+    security:
+        enabled: true
+        oauth2:
+            enabled: true
+            clients:
+                keycloak: 
+                    client-id: 'keycloak-client'
+                    openid:
+                        issuer: http://localhost:8090/auth/realms/Keycloak
+        intercept-url-map:
+        -
+            pattern: /graphiql
+            access:
+                - isAuthenticated()
```

Comment:
This will result in `Graphiql` stopping working because `Graphiql` doesn't allow to authenticate. Please add `TODO: fix or remove Graphiql`